doc/release-notes-1.9.rst

```diff
@@ -123,7 +124,9 @@ Documentation
 Tests and Samples
 *****************
 
-* change description
+* Added test Case to stress test round robin scheduling in schedule_api test.
```

Comment:
You could combine these into one bullet:

     * Added cases in schedule_api test to stress test round robin and priority scheduling.